src/example.ts

```diff
@@ -127,15 +127,18 @@ OT.getUserMedia()
 
 // Start publishing when you click the publish button
 publishBtn.addEventListener("click", () => {
+  const videoSource = videoSelector.options[videoSelector.selectedIndex].value;
+  const audioSource = audioSelector.options[audioSelector.selectedIndex].value;
+  console.log(videoSource, audioSource);
```

Comment:
Do we want to keep the log here or was this just for debug? If we want to keep it, suggest adding a little more context to the log line.

Reply:
Good catch I'll remove this